content/en/concepts/payment-methods.md

```diff
@@ -5,15 +5,32 @@ position: 10
 category: Payment Concepts
 ---
 
-Conteúdo
+Each payment method is defined by 3 key characteristics:
+- **Is it pull or push**: How the funds for the customer are acquired from the customer.
+- **Flow**: The type of action the user takes to make the payment.
+- **Asynchronous / Synchronous**: The charge happens immediately, or only after a delay. 
 
-## Credit / Debit Card
+## Pull or Push
+Each method of payment is categorized as either pull or push, depending on how funds are transferred from the customer’s payment method.
+
+- Using a pull method, you debit the funds from the customer’s account after the customer has provided consent. Card payments are an example of a pull method: your customer’s card is debited when a payment is made, and no customer interaction is required for subsequent debits.
 
-Isto aparece aqui
+- Using a push method, the customer sends the funds to you. Virtual IBAN or Multibanco is a good example.
+
+## Credit / Debit Card
+A Credit / Debit card is a payment card issued to users (cardholders) to enable the cardholder to pay a merchant good or service. This is a Synchronous pull payment method. This means that the payment happens immediately. Credit / Debit cards support also Authorizations where you can captivate an amount for later capture. 
```

Comment:
"support also" -> "also support"
Is "captivate" a used term in the industry?
Are we going with "Authori**z**ations" here even though the API uses "Authori**s**ations"?

Reply:
Changed both things. I also have doubts about the captivate word, it doesn't sound correct but I don't know the right one. Thoughts?